pages/treeherder.py

```diff
@@ -122,6 +122,10 @@ def unclassified_failure_count(self):
     def clear_filter(self):
         self.selenium.find_element(*self._clear_filter_locator).click()
 
+    def clear_filter_by_shortcut(self):
```

Comment:
It duplicates `clear_filter` function, but using keyboard shortcut.
What to do with this??